cmds/boot/uinit/main.go

```diff
@@ -42,14 +61,24 @@ func main() {
 
 	sleepInterval := time.Duration(*interval) * time.Second
 
-	if ipmi, err := ipmi.Open(0); err != nil {
+	ipmi, err := ipmi.Open(0)
```

Comment:
Should all the bmc communication be moved to a separate function?

Reply:
Updated PR.